zio-http/src/main/scala/zhttp/service/Handler.scala

```diff
@@ -16,40 +16,92 @@ private[zhttp] final case class Handler[R](
   runtime: HttpRuntime[R],
   config: Server.Config[R, Throwable],
   serverTimeGenerator: ServerTime,
-) extends SimpleChannelInboundHandler[FullHttpRequest](false)
+) extends ChannelInboundHandlerAdapter
     with WebSocketUpgrade[R]
     with ServerResponseHandler[R] { self =>
```

Comment:
```suggestion
    with ServerResponseHandler[R] { self =>
    val useObjectAggr: Boolean = config.objectAggregator < 0
```

Reply:
Done in the serverConfig and used it here